### PR TITLE
Add clang-11 docker configurations

### PR DESCRIPTION
**Pull request by veictry**
Based on nvbug https://nvbugs/200700358

added the required docker configuration related to the build with Clang-11

The image is based on Ubuntu 20.04.
In addition, we define DEBIAN_FRONTEND and TZ in docker layer.

### CODE REVIEW

File: docker/host_x86_64/ubuntu_20.04/target_x86_64/ubuntu_20.04/clang_11/cxx_11/base.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Dockerfile for libcudacxx_base:host_x86_64_ubuntu_20.04__target_x86_64_ubuntu_20.04__clang_10_cxx_11
```
Comment (griwes) [inline docker/host_x86_64/ubuntu_20.04/target_x86_64/ubuntu_20.04/clang_11/cxx_11/base.Dockerfile:1]: Please fix the comments to refer to Clang 11 and not 10. (Same for the other files.)

**Review by griwes — CHANGES_REQUESTED** [review]
The files have DOS line endings, please convert to Unix line endings.

Also, I tried running this with a compiler from the dev branch from today and it still says that clang 11 isn't a supported host compiler...

**Comment by griwes** [comment]
Actually ignore the compiler comment, seems my dev environment wasn't completely built, works after applying dos2unix. The rest of the comments stand.

File: docker/host_x86_64/ubuntu_20.04/target_x86_64/ubuntu_20.04/clang_11/cxx_11/base.Dockerfile (hunk 1 above)
Comment (veictry) [reply docker/host_x86_64/ubuntu_20.04/target_x86_64/ubuntu_20.04/clang_11/cxx_11/base.Dockerfile:1]: @griwes 
Thanks Michal!
I'll modify them right now

Comment (veictry) [reply docker/host_x86_64/ubuntu_20.04/target_x86_64/ubuntu_20.04/clang_11/cxx_11/base.Dockerfile]: @griwes 
Done

Comment (veictry) [reply docker/host_x86_64/ubuntu_20.04/target_x86_64/ubuntu_20.04/clang_11/cxx_11/base.Dockerfile]: For the converting from DOS line endings to Unix line endings, I'll try to fix it now.

Comment (veictry) [reply docker/host_x86_64/ubuntu_20.04/target_x86_64/ubuntu_20.04/clang_11/cxx_11/base.Dockerfile]: Done the last task. (submitted by my own Github account)

**Comment by griwes** [comment]
The variant_configuration.bash files still have windows line endings. Can you fix that and squash all the commits into a single one?

**Comment by veictry** [comment]
> The variant_configuration.bash files still have windows line endings. Can you fix that and squash all the commits into a single one?

The line ending issue has been fixed and the commit is modified.
Hi @griwes 
Please help to check it again. Thanks!

**Comment by veictry** [comment]
All windows line ending should be changed to unix line ending
```shell
chengjiew@chengjiew-nvidia:ubuntu_20.04$ git reset --hard
HEAD is now at 6000513 add clang-11 docker configurations
chengjiew@chengjiew-nvidia:ubuntu_20.04$ grep -lR $'\r'
chengjiew@chengjiew-nvidia:ubuntu_20.04$```

**Comment by veictry** [comment]
Hi @griwes 
My fixes are ready now. Please help to check again. Thanks!

**Pull request closed without merging**